python/decorators/decorators.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2022 Autodesk, Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the ShotGrid Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the ShotGrid Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Autodesk, Inc.
+
+from functools import wraps
+
+import sgtk
+from sgtk.platform.qt import QtGui, QtCore
+
+
+def wait_cursor(func):
+    """
+    Decorator function that overrides the Qt cursor to show the waiting cursor while a function executes.
```

Comment:
Could you split the line at 80 (or 120 I can't remember) chars like the rest of the code, please?